index.html

```diff
@@ -61,8 +61,231 @@ <h1 class="box1">
                </ul>
-           </div>
-        </main>
+            </section>
+            
+            <section>
+              <ul class="project_contents">
+                 <li class="project">
+                     <img src="images/project1.png" alt="Project Image 1">
+                     <div class="project_description">
+                        <h2 class="project_title">
+                           Tonic
+                        </h2>
+                        <div class="project_info">
+                          <p class="project_title_II">
+                            CANOPY
+                          </p>
+                          <ul class="project_job_title_info_container">
+                              <li class="job_title_info">
+                                Back End Dev
+                              </li>
+                              <li class="job_title_info">
+                                2015
+                              </li>
+                          </ul>
+                        </div>
+                        <p class="project_full_desc">
+                          A daily selection of privately personalized reads; no accounts or sign-ups required.
+                        </p>
+                        <div class="project_tags_container">
+                            <p class="project_tag">html</p>
+                            <p class="project_tag">css</p>
+                            <p class="project_tag">javaScript</p>
+                        </div>
+                        <button type="button" class="view_project_btn">
+                            See Project
+                        </button>
+                     </div>
+                 </li>
+                 <li class="project">
+                  <img src="images/project2.png" alt="Project Image 2">
+                  <div class="project_description">
+                     <h2 class="project_title">
+                        Multi-Post Stories
+                     </h2>
+                     <div class="project_info">
+                       <p class="project_title_II">
+                         CANOPY
+                       </p>
+                       <ul class="project_job_title_info_container">
+                           <li class="job_title_info">
+                             Back End Dev
+                           </li>
+                           <li class="job_title_info">
+                             2015
+                           </li>
+                       </ul>
+                     </div>
+                     <p class="project_full_desc">
+                       A daily selection of privately personalized reads; no accounts or sign-ups required.
+                     </p>
+                     <div class="project_tags_container">
+                         <p class="project_tag">html</p>
+                         <p class="project_tag">css</p>
+                         <p class="project_tag">javaScript</p>
```

Comment:
- [ ] Please let's make these project technologies a `ul` list and each of them a `li` instead of using a `p` tag. The reason for this is pointed above :point_up: :point_up: :point_up: